converters/toZigbee.js

```diff
@@ -6331,6 +6331,17 @@ const converters = {
             await entity.read('lightingBallastCfg', ['wiserControlMode'], {manufacturerCode: herdsman.Zcl.ManufacturerCode.SCHNEIDER});
         },
     },
+    wiser_motion_sensitivity: {
+        key: ['sensitivity'],
+        convertSet: async (entity, key, value, meta) => {
+            // Sensitivity levels are not documented so we cannot map to an enum
+            await entity.write('ssIasZone', {'currentZoneSensitivityLevel': value});
```

Comment:
What about merging this with TS0210_sensitivity? (https://github.com/Koenkk/zigbee-herdsman-converters/blob/8212f77eca0bcef157c31510a0e1d35fad5a87b3/converters/toZigbee.js#L6193)

Reply:
TS0210 seems to use 0,2,6 as the sensitivity levels, I am not sure what the levels exactly are on the Wiser (that's why there's no enum) but the range seems to be 0-3 so the options don't match with the TS0210. I don't have the schneider gateway unfortunately so reverse-engineering is a pain and also the sensor just ignores 99% of all commands sent to it even if I trigger them right after a report or reset / pairing.

Reply:
Although the levels don't match, we can still use it via numbers by changing `const sens = {'high': 0, 'medium': 2, 'low': 6}[value];` to `const sens = typeof value === 'string' ? {'high': 0, 'medium': 2, 'low': 6}[value] : value;`